app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="card">
+<div class="card-header">
+<h2>Log in</h2>
+</div>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field form-group">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field form-group">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field form-group">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+<div class="actions">
+<%= render "devise/shared/links" %>
+</div>
+</div>
```

Comment:
- [ ] Kindly improve the styling on the sign in page as well.

  ![image](https://github.com/Tracy-miranja/budget/assets/36542986/7302a8a1-97a3-4f44-be20-ee1e36047880)